reference/pkg/nodejs/@pulumi/aws/apigateway/index.md

```diff
@@ -1926,6 +1918,14 @@ public requestParameters: pulumi.Output<{ ... } | undefined>;
 
 
 A map of request query string parameters and headers that should be passed to the integration.
+For example:
+```hcl
```

Comment:
I didn't realize we include hcl... It might be nice at some point to detect & omit.

Reply:
This is due to https://github.com/pulumi/pulumi-terraform/issues/135 and related issues around how we generate these docs.